src/layout/DefaultLayout.tsx

```diff
@@ -1,40 +1,9 @@
 import { FunctionComponent } from 'react';
-import Link from 'next/link';
+import PublicHeader from '../components/PublicHeader';
```

Comment:
You've put this component in src/components, which I think is a great place to put it. But you put the `OrgHeader` component from #31 in src/layout.

You don't need to do anything about this now, but I think these two components are very similar and probably belong in the same top-level source directory. I suggest we start the next sprint by doing some spring cleaning, creating a more versatile directory structure and agreeing on some principles about what goes where. 🧹

Reply:
Oh yes, forgot to mention this, but I figured it would be messy to start moving unrelated files in the context of this issue. Spring cleaning sounds good!

Reply:
> Oh yes, forgot to mention this, but I figured it would be messy to start moving unrelated files in the context of this issue.

Great instinct, padawan! You are developing a programmer's mind. 👏